language/move-lang/tests/move_check/translated_ir_tests/signer/triple_signer.exp

```diff
@@ -0,0 +1,33 @@
+error: 
+
+    ┌── tests/move_check/translated_ir_tests/signer/triple_signer.move:13:9 ───
+    │
+ 13 │     fun t1(_s: &signer, _s2: &signer, u: u64, _s3: &signer) {
+    │         ^^ Invalid parameter for script function 't1'
+    ·
+ 13 │     fun t1(_s: &signer, _s2: &signer, u: u64, _s3: &signer) {
+    │                                                    ------- '&signer's must be a prefix of the arguments to a script
```

Comment:
Might be worth explaining this more. Feels maybe a tad compilery
`"'&signer's must be a prefix of the script arguments--they must come before any non-signer types"`